src/ZfcDatagrid/Renderer/BootstrapTable/View/Helper/TableRowFactory.php

```diff
@@ -1,21 +1,22 @@
 <?php
 namespace ZfcDatagrid\Renderer\BootstrapTable\View\Helper;
 
+use Zend\ServiceManager\AbstractPluginManager;
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
 class TableRowFactory implements FactoryInterface
 {
     /**
-     * @param  ServiceLocatorInterface $serviceLocator
+     * @param  ServiceLocatorInterface|AbstractPluginManager $serviceLocator
```

Comment:
in the method signatur there is a check for `ServiceLocatorInterface` so why you added `AbstractPluginManager` in the method documentation?